frontend/src/components/route/RouteDetails.js

```diff
@@ -29,10 +29,12 @@ export default function RouteDetails() {
             <PageLayout>
                 <Header headerText={route?.routeName}/>
                 <ul>
+                    <div className={"placeholder"}/>
                     {!legs? null :
                         legs.map((leg, index) => <Leg key={leg.legId} leg={leg} index={index}/>)
                     }
                     <RouteEnd endWaypoint={endWaypoint}/>
+                    <div className={"placeholder"}/>
```

Comment:
Semantically only `li` are allowed in `ul`. if you need space at the bottom, select the last child and add a `::after` pseudo element ;)